api/handlers.go

```diff
@@ -13,24 +14,39 @@ type Response struct {
 	code    int
 }
 
-// GetSnippet - Handler gets a single code snippet from service by its id
+const (
+	statusError = "error"
+	statusOK = "ok"
```

Comment:
Yep didn't realise this before, but yes just checked a few other open source packages and your right.